api/geth_backend.go

```diff
@@ -400,6 +400,60 @@ func (b *GethStatusBackend) StartNodeWithAccount(acc multiaccounts.Account, pass
 	return err
 }
 
+func (b *GethStatusBackend) ExportUnencryptedDatabase(acc multiaccounts.Account, password, directory string) error {
+	b.mu.Lock()
+	defer b.mu.Unlock()
+	if b.appDB != nil {
+		return nil
+	}
+	if len(b.rootDataDir) == 0 {
+		return errors.New("root datadir wasn't provided")
+	}
+
+	// Migrate file path to fix issue https://github.com/status-im/status-go/issues/2027
+	oldPath := filepath.Join(b.rootDataDir, fmt.Sprintf("app-%x.sql", acc.KeyUID))
+	newPath := filepath.Join(b.rootDataDir, fmt.Sprintf("%s.db", acc.KeyUID))
```

Comment:
Do we need to pass in a whole `multiaccounts.Account` if we only use the `acc.KeyUID`?